[2023-03-16T09:56:23.203+0000] {taskinstance.py:1084} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: dag_with_python_operator_v6.get_name manual__2023-03-16T09:56:22.468522+00:00 [queued]>
[2023-03-16T09:56:23.217+0000] {taskinstance.py:1084} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: dag_with_python_operator_v6.get_name manual__2023-03-16T09:56:22.468522+00:00 [queued]>
[2023-03-16T09:56:23.217+0000] {taskinstance.py:1282} INFO - 
--------------------------------------------------------------------------------
[2023-03-16T09:56:23.218+0000] {taskinstance.py:1283} INFO - Starting attempt 1 of 6
[2023-03-16T09:56:23.218+0000] {taskinstance.py:1284} INFO - 
--------------------------------------------------------------------------------
[2023-03-16T09:56:23.236+0000] {taskinstance.py:1303} INFO - Executing <Task(PythonOperator): get_name> on 2023-03-16 09:56:22.468522+00:00
[2023-03-16T09:56:23.240+0000] {standard_task_runner.py:55} INFO - Started process 1844 to run task
[2023-03-16T09:56:23.246+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'dag_with_python_operator_v6', 'get_name', 'manual__2023-03-16T09:56:22.468522+00:00', '--job-id', '45', '--raw', '--subdir', 'DAGS_FOLDER/python_operator.py', '--cfg-path', '/tmp/tmpzlxfankx']
[2023-03-16T09:56:23.248+0000] {standard_task_runner.py:83} INFO - Job 45: Subtask get_name
[2023-03-16T09:56:23.327+0000] {task_command.py:388} INFO - Running <TaskInstance: dag_with_python_operator_v6.get_name manual__2023-03-16T09:56:22.468522+00:00 [running]> on host c3799dfdd69d
[2023-03-16T09:56:23.408+0000] {taskinstance.py:1511} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Mr.Phi
AIRFLOW_CTX_DAG_ID=dag_with_python_operator_v6
AIRFLOW_CTX_TASK_ID=get_name
AIRFLOW_CTX_EXECUTION_DATE=2023-03-16T09:56:22.468522+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-03-16T09:56:22.468522+00:00
[2023-03-16T09:56:23.433+0000] {python.py:177} INFO - Done. Returned value was: None
[2023-03-16T09:56:23.444+0000] {taskinstance.py:1326} INFO - Marking task as SUCCESS. dag_id=dag_with_python_operator_v6, task_id=get_name, execution_date=20230316T095622, start_date=20230316T095623, end_date=20230316T095623
[2023-03-16T09:56:23.459+0000] {local_task_job.py:212} INFO - Task exited with return code 0
[2023-03-16T09:56:23.494+0000] {taskinstance.py:2585} INFO - 0 downstream tasks scheduled from follow-on schedule check
